internal/handler/tgmessagehandler/util.go

```diff
@@ -0,0 +1,190 @@
+package tgmessagehandler
+
+import (
+	"backend/internal/api"
+	"backend/internal/dataaccess/chat"
+	"backend/internal/dataaccess/message"
+	"backend/internal/dataaccess/requestquery"
+	"backend/internal/model"
+	"backend/internal/viewmodel"
+	"backend/internal/ws"
+	"backend/pkg/error/internalerror"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math/rand"
+	"time"
+
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api/v5"
+	"gorm.io/gorm"
+)
+
+var (
+	NoQueryFoundResponse = fmt.Sprintf(
+		"Please start a new query first. Use /%s, or /%s",
+		QueryCmdWord, RequestCmdWord,
+	)
+	NoQueryFoundErr = internalerror.RequestQueryNotFoundError{}
+	NoChatFoundErr  = internalerror.ChatNotFoundError{}
+)
+
+// TODO: Implement AI response
+func getAICategorisation(_ *gorm.DB, _ string) (model.Type, error) {
+	decision := rand.Intn(3)
+	switch decision {
+	case 0:
+		return model.TypeQuery, nil
+	case 1:
+		return model.TypeRequest, nil
+	default:
+		return model.TypeUnknown, nil
```

Comment:
note: from my understanding, i think the ai does not return this response

i added this as i thought a zero value may be convenient for implementation